package.json

```diff
@@ -197,5 +197,8 @@
     "vitest-fail-on-console": "0.7.1",
     "webpack": "5.94.0",
     "zod": "3.23.8"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-react": "4.3.4"
   }
```

Comment:
The root package doesn't have any dev dependencies, so we can simplify by including everything in the dependencies.

Reply:
Kind of splitting hairs, but since the root is never published, it doesn't have production dependencies. I would really recommend swapping all of them to `devDependencies` for clarity.

I'll swap this to production dependencies for now though for consistency.